docs/source/_static/js/installer_widget.js

```diff
@@ -55,6 +55,26 @@ your custom Environments to the MicroK8s node.
 directly.</p>
           `,
       },
+      K3s: {
+        instructions: `
+<p>First, deploy <a href="https://k3s.io/">K3s</a> without having traefik installed by running
```

Comment:
```suggestion
<p>First, install <a href="https://k3s.io/">K3s</a> without deploying traefik by running
```

Might be even better to make traefik into `traefik` as well (or a hyperlink).